cmd/cli/app/get.go

```diff
@@ -38,7 +39,8 @@ type getCmdOutput struct {
 
 func (g *getCmdOutput) String() string {
 	s := fmt.Sprintf("Index: %d\n", g.LogIndex)
-	s += fmt.Sprintf("IntegratedTime: %d\n", g.IntegratedTime)
+	dt := time.Unix(g.IntegratedTime, 0).UTC().Format(time.RFC822)
```

Comment:
personally, I would do RFC3339 but either way its better than Unix epochs.

Reply:
Whoops, I had that but switched it for some reason.